### PR TITLE
[AN-5120] displays the CBR text if the other party uses CBR, 

### PR DESCRIPTION
**Pull request by makingthematrix**
Also removes the text from video calls.
#### APK
[Download build #8688](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8688/artifact/build/artifact/wire-dev-PR729-8688.apk)
[Download build #8689](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8689/artifact/build/artifact/wire-dev-PR729-8689.apk)
[Download build #8690](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8690/artifact/build/artifact/wire-dev-PR729-8690.apk)
[Download build #8691](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8691/artifact/build/artifact/wire-dev-PR729-8691.apk)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -55,7 +55,7 @@ task licenseFormatAndroid(type: nl.javadude.gradle.plugins.license.License) {
 licenseFormat.dependsOn licenseFormatAndroid
 
 ext {
-    majorVersion = "2.30."
```
Comment (hakonbo) [inline build.gradle]: I guess you branched off master, should have used rel/2.30.x, please revert major version to 2.30

File: build.gradle [hunk 2]
```diff
@@ -73,7 +73,7 @@ ext {
     playServicesVersion = '10.0.1'
 
     audioVersion = System.getenv("AUDIO_VERSION") ?: '1.209.0@aar'
-    zMessagingDevVersionBase = '95.1422'
+    zMessagingDevVersionBase = '96.1426'
     zMessagingDevVersion = "${zMessagingDevVersionBase}@aar"
     // Release version number must be like this X.0(.Y)
     zMessagingReleaseVersion = System.getenv("ZMESSAGING_VERSION") ?: '95.1.336@aar'
```
Comment (hakonbo) [inline build.gradle]: Can you update release version as well, please?

File: build.gradle (hunk 1 above)
Comment (makingthematrix) [inline build.gradle]: ok, done

**Comment by makingthematrix** [comment]
Build again

**Comment by red-avtovo** [comment]
💎 👍

**Pull request merged**